zipkin-common/src/test/scala/com/twitter/zipkin/storage/DependencyStoreSpec.scala

```diff
@@ -103,28 +103,25 @@ abstract class DependencyStoreSpec extends JUnitSuite with Matchers {
     val two = Endpoint(127 << 24 | 2, 9410, "trace-producer-two")
     val three = Endpoint(127 << 24 | 3, 9410, "trace-producer-three")
 
-    val trace = List(
-      Span(10L, "get", 10L, None, Some(1445136539256150L), Some(1152579L), List(
```

Comment:
overall question. Are we saying that the spark job cannot aggregate data earlier than today? One possibly un-obvious aspect of this test is that it is explicitly older than *today*. Ex. it is easy to accidentally create an impl that cannot re-process historical spans.

If these stamps were redone to be "last week" or "3 days ago" I think it could call out the intent more explicitly.